File: Architecture.Core/Entities/ProfIncomeInfo.cs
using System;
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;

namespace Architecture.Core.Entities
{
    public class ProfIncomeInfo : Auditable
    {
        public int IncomeInfoId { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public int? IncomeTypeId { get; set; }
        public virtual IncomeType IncomeType { get; set; }
        public decimal YearlyIncome { get; set; }
        public decimal MonthlyIncome { get; set; }        
        public string Year { get; set; }
        public string Month { get; set; }
        public string Document { get; set; }
        public string Status { get; set; }

    }
}

File: Architecture.Web/ClientApp/src/app/Shared/Entity/ClientProfile/profIncomeInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class profIncomeInfo extends Auditable {
    public  IncomeInfoId: number;
    public profileId: number;
    //public virtual ProfBasicInfo ProfBasicInfo { get; set; }

        //public int? IncomeTypeId { get; set; }
        //public virtual IncomeType IncomeType { get; set; }

    public incomeTypeId: number;

    public yearlyIncome: number;

    public monthlyIncome: number;

    public year: Date;

    public month: Date;

    public document: String;

    public status: String;
        
}


